jbx-protocol/juice-interface
# V2 splits + fixes to V2 units

## What does this PR do and why?

Fixes the way we create a display splits for V2 projects. I've left comments throughout to explain the big rocks.

Also fixes (I think) the units for every other value in V2 (reserved rate, discount rate, etc.). There's still some testing we'll have to do (like making sure payout recipients actually get the right amount of funds), but think we need to get this in ASAP to save us some sanity.

## How to test

You can test project creation to make sure you end up with the same splits as you defined.

Also try viewing `/#/v2/p/47`.

## Acceptance checklist
